### ISSUE TITLE
Library errors  with haxe version 4.0 preview 2 

### ISSUE THREAD

**Issue by PXshadow** [HaxeFoundation/record-macros#15]
Errors from RecordMacros
```
D:/haxe/lib/record-macros/1,0,0-alpha/src/sys/db/RecordMacros.hx:736: characters 12-14 : Unmatched patterns: OpIn
D:/haxe/lib/record-macros/1,0,0-alpha/src/sys/db/RecordMacros.hx:902: characters 8-17 : Not enough arguments
D:/haxe/lib/record-macros/1,0,0-alpha/src/sys/db/Manager.hx:31: characters 13-19 : Build failure
```

**Comment by jonasmalacofilho** [comment]
From what I can tell, this was most likely fixed in #12.

Can you try the git version?

```
haxelib git record-macros https://github.com/HaxeFoundation/record-macros
```

**Comment by PXshadow** [comment]
Worked perfectly thank you so much!
